gc/base/standard/Scavenger.cpp

```diff
@@ -298,6 +298,11 @@ MM_Scavenger::initialize(MM_EnvironmentBase *env)
 		return false;
 	}
 
+#if defined(OMR_GC_MODRON_SCAVENGER)
+	/* register the scavenger in GCExtensionsBase */
+	_extensions->registerScavenger(this);
+#endif /* OMR_GC_MODRON_SCAVENGER */
+
```

Comment:
minor difference, but i'd keep it at original spot (higher level entity knows better to whom it should be registered)

comment is redundant (the method is self explanatory)